tests/unittests/DeviceManagerTest.cpp

```diff
@@ -392,6 +392,8 @@ TEST_P(DeviceManagerTest, ReuseModule) {
   EXPECT_NE(ctx1, ctx2);
 }
 
+#ifdef GLOW_WITH_CPU
```

Comment:
slightly better fix is to call `createDeviceManager` with the backend kind on line 400

Reply:
Well there is no `DeviceManager::maxMemoryBytes_` -- it's only in the Interpreter and CPUBackend. And it needs to be set to 1. So I don't think that would work.

Also the way the test is currently formulated I'm surprised passes. I think there needs to be better error handling here. It looks like we're explicitly creating a `CPUDeviceManager`, but then calling `addNetwork()` on it with a `InterpreterFunction`, and yet the test passes...?

Reply:
If all device managers expect to have maxMemoryBytes supplied as part of the init we should extend createDeviceManager call with that param IMO.